web/src/components/CityNotCooperatingFooter.tsx

```diff
@@ -26,7 +26,7 @@ const StyledIcon = styled(Icon)`
 const Button = styled(Link)`
   background-color: ${props => props.theme.colors.themeColor};
   text-decoration: none;
-  color: ${props => props.theme.colors.textColor};
+  color: ${props => (props.theme.isContrastTheme ? props.theme.colors.backgroundColor : props.theme.colors.textColor)};
```

Comment:
❓ This looks confusing to me, why would the color of the button text be a background color? Can we give that color a different name?

Reply:
Hmm, the problem here is basically that the text color for the contrast theme (white) has too little contrast on the theme color (yellow). Therefore, we set the text color to the background color here. This is basically only a workaround which should be history with the switch to the new design system. I therefore think that the effort to rename this would be too much.

Reply:
@lunars97 using a helper as proposed in another comment by me for this would clarify and make this more readable imo. Still optional though.